### PR TITLE
api-server: configure default mixin sources

### PR DESCRIPTION
**Pull request by bajtos**
As a follow-up for https://github.com/strongloop/loopback-boot/pull/131, this patch modifies the scaffolded `model-config.json` to include lookup paths for mixins.

 - loopback/common/mixins
 - loopback/server/mixins
 - ../common/mixins
 - ./mixins

Connect to strongloop/loopback-boot#79

/to @ritch please review
/cc @PradnyaBaviskar

### CODE REVIEW

File: templates/components/api-server/component.js [hunk 1]
```diff
@@ -44,6 +44,12 @@ template.server = {
         'loopback/server/models',
         '../common/models',
         './models',
+      ],
+      mixins: [
```
Comment (bajtos) [inline templates/components/api-server/component.js]: I am pondering whether "mixins" is a good key name, perhaps "mixinSources" would be a better name? (See also "sources" key used for model sources above.)

**Comment by bajtos** [comment]
@ritch ping

File: templates/components/api-server/component.js (hunk 1 above)
Comment (ritch) [inline templates/components/api-server/component.js]: It should be consistent with what we have for models IMO

**Comment by ritch** [comment]
LGTM

File: templates/components/api-server/component.js (hunk 1 above)
Comment (bajtos) [inline templates/components/api-server/component.js]: > It should be consistent with what we have for models IMO

:+1: for consistency

I am not sure how to apply that rule in this particular case. What property name would you find more consistent?

This is the current prologue of the scaffolded `model-config.json` file:

```json
{
  "_meta": {
    "sources": ["loopback/common/models", "etc."],
    "mixins": ["loopback/common/mixins", "etc."]
}
```

**Pull request merged**